mathics/core/pattern.py

```diff
@@ -289,27 +289,28 @@ def per_expr(yield_expr, expr_groups, sum=0):
 
                         if expr_groups:
                             expr, count = expr_groups.popitem()
-                            max_per_pattern = count / len(patterns)
+                            max_per_pattern = int(count / len(patterns))
```

Comment:
Again, this isn't related to PR.

Reply:
> Again, this isn't related to PR.

This was addressed in https://github.com/mathics/Mathics/pull/937. Instead of `int(... / ...)`  `//` was used.